util/cron/nightly

```diff
@@ -710,6 +714,15 @@ if ($runtests == 0) {
     }
 
     $ENV{'CHPL_HOME'} = $chplhomedir;
+
+    # Confirm Chapel built correctly before start_test / paratest
+    if ($buildcheck eq 1) {
+        $buildcheckcommand = "bash ../test/checkChplInstall";
```

Comment:
I think we should literally call `make check` instead of the underlying script that gets called. 

Maybe replacing these 4 lines with something ike:

```
mysystem("cd $CHPL_HOME && make check", "running `make check`", 1, 1, 1);
```

^^ 
1st arg is the command to run
2nd is the message used on failure so it will say "Error running `make check`: $status"
3rd is whether a failure of this command is fatal (I think it is in this case)
4th is whether a mail will be sent out on failure (I think we need to send one)
5th is whether to print out the command being executed (replaces your 'print "Executing ..."';)